src/main/webapp/app/text-assessment/text-assessment.component.ts

```diff
@@ -42,8 +43,8 @@ export class TextAssessmentComponent implements OnInit, OnDestroy, AfterViewInit
     isAtLeastInstructor = false;
     busy = true;
     showResult = true;
-    hasComplaint = false;
     complaint: Complaint;
+    complaintTypes = ComplaintType;
```

Comment:
I think it would be more consistent to keep it as `ComplaintType` instead of `complaintTypes`.